### PR TITLE
created occupational-therapy for 5 students

### PR DESCRIPTION
**Pull request by GytisPht**

### CODE REVIEW

**Comment by GytisPht** [comment]
done

File: js/data/med/occupational-therapy.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
+import StudyProgram from '../../entities/study-program.js';
+import StudyProgramSemester from '../../entities/study-program-semester.js';
+import Module from '../../entities/module.js';
+import { medicineFaculty } from '../faculties.js';
+
+
```
Comment (ZilvinasVidmantas) [inline js/data/med/occupational-therapy.js:6]: No empty lines

File: js/data/med/occupational-therapy.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+occupationalTherapySemester7module1,
+occupationalTherapySemester7module2,
+occupationalTherapySemester7module3,
+occupationalTherapySemester7module4,
+
+occupationalTherapySemester8module1,
+occupationalTherapySemester8module2,
+];
+
+export default occupationalTherapy;
```
Comment (ZilvinasVidmantas) [inline js/data/med/occupational-therapy.js:149]: misson EOL

File: js/data/modules.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { dataScienceModules } from './info-math/data-science.js';
 import { economicsAndBusinessModules } from './eco/business-law.js';
 import { nanoChemistryModules } from './chem/nano-chemistry.js';
 import { odontologyModules } from './med/odontology.js';
+import { occupationalTherapy } from './info/software-systems.js';
```
Comment (ZilvinasVidmantas) [inline js/data/modules.js:7]: impoer order inconssistent

File: js/main.js [hunk 4]
```diff
@@ -1,10 +1,12 @@
+
 import Student from './entities/student.js';
 import softwareSystems from './data/info-math/software-systems.js';
 import dataScience from './data/info-math/data-science.js';
 import mathematicsAndItsApplications from './data/info-math/mathematics-and-its-applications.js';
 import businessAndLaw from './data/eco/business-law.js';
 import nanoChemistry from './data/chem/nano-chemistry.js';
 import odontology from './data/med/odontology.js';
+import occupationalTherapy from './data/med/occupational-therapy.js';
```
Comment (ZilvinasVidmantas) [inline js/main.js:9]: your import must be last

**Review by ZilvinasVidmantas — CHANGES_REQUESTED** [review]

**Pull request merged**